index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+      <h1 class="header_name">Nicholas</h1>
+      <nav class="header_menu">
+        <a href=""><img src="image/Union.png" alt="Union image" /></a>
+      </nav>
+    </header>
+    <section class="headline">
+      <h2 class="title_1">I’m Nicholas Glad to see you!</h2>
+      <p class="para_1">
+        I’m a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <h3 class="title_2">LET’S CONNECT</h3>
+      <ul class="social-icons">
+        <li><a href="https://twitter.com/MysticalAmissah"><img src="./image\twitter.png" alt="twitter"></a></li>
+        <li><a href="https://www.linkedin.com/in/nicholas-amissah-153b09154"><img src="./image\Linkedin icon.png" alt="Linkedin"></a></li>
+        <li><a href="#"><img src="./image\messenger.png" alt="messenger"></a></li>
+        <li><a href="https://github.com/atok624"><img src="./image\github.png" alt="github"></a></li>
+        <li><a href="#"><img src="./image\hands.png" alt="hands"></a></li>
+      </ul>
+    </section>
```

Comment:
The left-bottom border in the Figma template is rounded and it is missing in your case. Kindly give an appropriate border-radius to the left bottom border of your section to match it with the Figma template.
| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/212347848-df04fa95-56d7-48d5-a2a3-d2db1f6bbc2d.png) | ![image](https://user-images.githubusercontent.com/33268927/212347781-30e6a077-89f1-4e14-ba89-f2a490bf5905.png) |